code/modules/client/preference_setup/loadout/loadout_gloves.dm

```diff
@@ -70,4 +82,29 @@
 
 /datum/gear/gloves/circuitry
 	display_name = "gloves, circuitry (empty)"
-	path = /obj/item/clothing/gloves/circuitry
\ No newline at end of file
+	path = /obj/item/clothing/gloves/circuitry
+
+/datum/gear/gloves/disposable
+	display_name = "gloves, disposable (selection)"
+	path = /obj/item/clothing/gloves/latex
+	cost = 2
+
+/datum/gear/gloves/disposable/New()
+	..()
+	var/dispos_gloves = list()
+	dispos_gloves["latex gloves"] = /obj/item/clothing/gloves/latex
+	dispos_gloves["nitrile gloves"] = /obj/item/clothing/gloves/latex/nitrile
+	gear_tweaks += new/datum/gear_tweak/path(dispos_gloves)
+
+//I added the rest of security to the allowed_roles because:
+//as much as I don't want to encourage the rest of security to do the investigators' work,
+//I think it wouldn't make much sense to have them not be able to provide the aid they can in HANDLING physical evidence--though feel free to disagree
+/datum/gear/gloves/forensic
+	display_name = "gloves, forensic"
+	path = /obj/item/clothing/gloves/forensic
+	allowed_roles = list("Security Officer", "Head of Security", "Warden", "Security Cadet", "Detective", "Forensic Technician")
+
+/datum/gear/gloves/duty
+	display_name = "gloves, work"
```

Comment:
Perhaps it might benefit from renaming the type path to '../work'?
Work gloves are a form of safety equipment which help provide protection against certain forms of heavy/indstural labour, as they usually come cushioned and padded. They're made of leather, but these are more for mechanical instruments rather than electrical, as I understand it.
Funnily enough, they're pretty easy to get a hold of and bring around with you—at least in the places I go to—but I suppose this point isn't going to be entirely relevant.